crates/kitsune_p2p/kitsune_p2p/src/test_util/switchboard/tests.rs

```diff
@@ -240,9 +239,7 @@ async fn sharded_4way_recent() {
 
 #[tokio::test(flavor = "multi_thread")]
 async fn sharded_4way_historical() {
-    observability::test_run().ok();
-    let mut u = arbitrary::Unstructured::new(&crate::NOISE);
-
+    // observability::test_run().ok();
```

Comment:
what's the implication of this being removed?

Reply:
No logging. We only really need logging when modifying/debugging a test, in which case someone can uncomment this line.

Reply:
FYI logging is only turned on when the environment variable is set so this is a little redundant

Reply:
True, in my IDE the env var (RUST_LOG) is set when running tests from the IDE though